### DISCUSSION TITLE
Padronização de commits e branchs

### DISCUSSION THREAD

**Discussion by LuanFreitasRibeiro** [andrebaltieri/AgendaEventos#64] [General]
Time,

Gostaria de abordar com todos a respeito do padrão de nomes para commits e branchs do nosso projeto, chegamos a um momento que iremos necessitar ter um padrão para não ficar bagunçado.

Estava pensando em aderir ao inglês para esses commits e branchs.

**Comment by jenrodrigues** [comment]
Por mim nao ha problemas fazermos tudo em Ingles.

**Comment by GabrielNalin** [comment]
Também acho melhor em ingles.

Em ter, 25 de mai de 2021 12:35, Luan Freitas ***@***.***>
escreveu:

**Comment by renebentes** [comment]
Terei um pouco de dificuldade, mas vou aderir ao que a maioria decidir
